### PR TITLE
Fix go coverage builds

### PR DESCRIPTION
**Pull request by inferno-chromium**

### CODE REVIEW

**Comment by inferno-chromium** [comment]
@catenacyber fyi

**Review by jonathanmetzman — APPROVED** [review]
LGTM

File: infra/base-images/base-builder-go/Dockerfile [hunk 1]
```diff
@@ -24,3 +24,6 @@ ENV GOPATH /root/go
 ENV PATH $PATH:/root/.go/bin:$GOPATH/bin
 
 RUN install_go.sh
+
+COPY ossfuzz_coverage_runner.go $GOPATH
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply infra/base-images/base-builder-go/Dockerfile:28]: Wait. Where is this file located. Is it located in base-builder-go or base-builder-new?

**Review by jonathanmetzman — CHANGES_REQUESTED** [review]

**Comment by inferno-chromium** [comment]
that file is in infra/base-images/base-builder-go/ossfuzz_coverage_runner.go

**Comment by inferno-chromium** [comment]
TBR to unblock broken go coverage builds.

**Pull request merged**